extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -1,623 +1,668 @@
-function getThingTypeUID(thingUID) {
```

Comment:
It seems you already applied the auto code formatter on the js files - please revert this and save your changes without reformatting, since https://github.com/eclipse/smarthome/pull/1015 is still open.